reason-react-native/src/apis/Event.md

```diff
@@ -1,12 +1,17 @@
 ---
 id: apis/Event
 title: Event
-wip: true
+officialDoc: https://github.com/facebook/react-native/blob/master/Libraries/Types/CoreEventTypes.js
 ---
 
-```reason
-// see https://github.com/facebook/react-native/blob/master/Libraries/Types/CoreEventTypes.js
+These are the types of objects returned in `Event` callbacks. Types are
+instances of `syntheticEvent('a)` and `responderSyntheticEvent('a)`.
+
+## Prametrised Types
```

Comment:
Parameterized? (it feels weid to me & I was thinking "Generic", but "parameters" & "parameterized" seems what's used in reasonml docs https://reasonml.github.io/docs/en/more-on-type#type-argument

Reply:
Doesn't feel weird to me, but I had to think about what to call these, so I checked the documentation.